src/lib/prover/prover.ml

```diff
@@ -167,15 +183,16 @@ module Functions = struct
         `Initialized )
 
   let extend_blockchain =
-    create Extend_blockchain_input.bin_t
-      [%bin_type_class: Blockchain.t Or_error.t]
+    create Extend_blockchain_input.Stable.Latest.bin_t
```

Comment:
Marking this because I think this (and the next change) are fine, if a tad unsettling.